frontend/src/components/layout/navigation/Navigation.tsx

```diff
@@ -0,0 +1,92 @@
+import Link from "next/link";
+import { useRouter } from "next/router";
+import { useLocalization } from "@fluent/react";
+import styles from "./Navigation.module.scss";
+import { SignUpButton } from "./SignUpButton";
+import { SignInButton } from "./SignInButton";
+import { ProfileData } from "../../../hooks/api/profile";
+import { UpgradeButton } from "./UpgradeButton";
+import { WhatsNewMenu } from "./whatsnew/WhatsNewMenu";
+import { UserMenu } from "./UserMenu";
+import { AppPicker } from "./AppPicker";
+import { MenuToggle } from "./MenuToggle";
+
+export type Props = {
+  theme: "free" | "premium";
+  hasPremium: boolean;
+  mobileMenuExpanded: boolean | undefined;
+  handleToggle: CallableFunction;
+  profile: ProfileData | undefined;
+  isLoggedIn: boolean;
+};
+/** Switch between the different pages of the Relay website. */
+export const Navigation = (props: Props) => {
+  const { l10n } = useLocalization();
+  const router = useRouter();
+  const {
+    theme,
+    mobileMenuExpanded,
+    handleToggle,
+    profile,
+    hasPremium,
+    isLoggedIn,
+  } = props;
+  const homePath = isLoggedIn ? "/accounts/profile" : "/";
+
+  const ToggleButton = () => (
+    <button className={styles["menu-toggle"]} onClick={() => handleToggle()}>
+      {/* passing toggle state to show correct icon */}
+      <MenuToggle toggleState={mobileMenuExpanded} />
+    </button>
+  );
+
+  return (
+    <nav aria-label={l10n.getString("nav-menu")} className={styles["site-nav"]}>
+      <Link href={homePath}>
+        <a
+          className={`${styles.link} ${styles["home-link"]} 
+          ${styles["hidden-mobile"]} 
+          ${router.pathname === homePath ? styles["is-active"] : null}`}
```

Comment:
Nitpick, but this `null` gets stringified and thus results in a class like `Navigation_hidden-mobile__EPCna null`:

```suggestion
          ${router.pathname === homePath ? styles["is-active"] : ""}`}
```

(Likewise below, for the FAQ link.)